src/gt4py/next/ffront/decorator.py

```diff
@@ -310,7 +310,7 @@ def __call__(
 
 
 try:
-    from gt4py.next.program_processors.runners.dace_iterator import Program
+    from gt4py.next.program_processors.runners.dace_fieldview.program import Program
```

Comment:
Since `dace_iterator.Program` is no longer used nor tested, you could delete that code.

Reply:
👍